Tools/cases_generator/generate_cases.py

```diff
@@ -276,11 +277,13 @@ def fromInstruction(instr: "AnyInstruction"):
             HAS_CONST_FLAG=variable_used(instr, "FRAME_CO_CONSTS"),
             HAS_NAME_FLAG=variable_used(instr, "FRAME_CO_NAMES"),
             HAS_JUMP_FLAG=variable_used(instr, "JUMPBY"),
+            HAS_LOCAL_FLAG=variable_used(instr, "GETLOCAL") or
+                           variable_used(instr, "SETLOCAL"),
         )
 
     @staticmethod
     def newEmpty():
-        return InstructionFlags(False, False, False, False)
+        return InstructionFlags(*(5 * (False,)))
```

Comment:
I liked the original way better, even if it becomes repetitive. I momentarily thought that the leading '*' was a syntax error.